syntaxes/cshell.tmLanguage.json

```
@@ -20,15 +20,15 @@
     "keywords": {
       "patterns": [
         {
-          "match": "(?<=^|;|&|\\s)(if|elif|while|until)(?=\\()",
+          "match": "(?<=^|;|&|\\s)(if|elif|else if|while|until)(?=.*\\()",
           "name": "keyword.control.shell"
         },
         {
-          "match": "(?<=^|;|&|\\s)(setenv|@)(?=\\s|;|&|$)",
+          "match": "(?<=^|;|&|\\s)(set|setenv|@)(?=\\s|;|&|$)",
```

```suggestion
          "match": "(?<=^|;|&|\\s)(setenv|@)(?=\\s|;|&|$)",
```

This is also handled by VSCode internally: https://github.com/microsoft/vscode/blob/main/extensions/shellscript/syntaxes/shell-unix-bash.tmLanguage.json#L1208:~:text=set

---

Ok. I remove this. See #2.